examples/ensemble/plot_voting_decision_regions.py

```diff
@@ -1,7 +1,7 @@
 """
-==================================================
-Plot the decision boundaries of a VotingClassifier
-==================================================
+=====================================================
+Visualizing class probabilities in a VotingClassifier
+=====================================================
```

Comment:
"class probabilities" is ambiguous. Here, those are the predictions of a given model that we visualize.

```suggestion
===============================================================
Visualizing the probabilistic predictions of a VotingClassifier
===============================================================
```

They can be interpreted as class probabilities only if the model predictions match the true data generating process (by achieving the lowest possible log-loss or Brier score).